aikau/src/main/resources/alfresco/menus/AlfMenuBarPopup.js

```diff
@@ -109,32 +109,51 @@ define(["dojo/_base/declare",
          {
             this.title = this.message(this.title);
          }
+         if (!this.iconAltText && this.label)
+         {
+            this.iconAltText = this.label;
+         }
+         else
+         {
+            this.iconAltText = this.message(this.iconAltText);
+         }
```

Comment:
It's not obvious from how Git shows the diff, but this.label has already been "messaged" directly before, so if it's available it's the messaged form.